ui/css/module/_styleguide.scss

```diff
@@ -0,0 +1,26 @@
+.styleguide-container {
+  padding: 1rem;
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.styleguide-example {
+  border: 1px solid $color-lt-gray;
+  padding: 1rem 2.5rem;
+
+  h4 {
+    text-transform: uppercase;
+    color: $color-gray;
+    letter-spacing: 1px;
+    font-size: 1em;
+  }
+}
+
+.example-title {
+  margin-top: 2rem;
+  margin-bottom: 1rem;
```

Comment:
Rather than defining everything in constant rems, could we use the new `$spacing` variables?